### PR TITLE
add Bislam's hackbar

### PR DESCRIPTION
**Pull request by concentratedloss**
Hey

### CODE REVIEW

File: _hackbars/bislam.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+layout: hackbar
+title: Hey
+author: Bislam S
+---
```
Comment (tasnimzotder) [inline _hackbars/bislam.md:5]:
```suggestion
---
layout: hackbar
title: Hey
author: Bislam S
---
```

File: _hackbars/bislam.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+<!-- Let's become a better developers so we can figure to stop our planet from polution-->
```
Comment (tasnimzotder) [inline _hackbars/bislam.md]: hey @dtf1337, you need to remove the comment -
```suggestion
Let's become a better developers so we can figure to stop our planet from polution
```

File: _hackbars/bislam.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```
Comment (tasnimzotder) [inline _hackbars/bislam.md]: you need to add your __MESSAGE__ on climate change

File: _hackbars/bislam.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Hey
+author: Bislam S
+---
+
+Let's become a better developers so we can figure to stop our planet from polution
```
Comment (concentratedloss) [inline _hackbars/bislam.md:7]: <----Let's become a better developers so we can figure to stop our planet from polution--->

**Comment by tasnimzotder** [comment]
hey @dtf1337, the remaining edit is your __MESSAGE__. You can add your __MESSAGE__ on climate change on your `.md` file.
File `_hackbars/bislam.md` line 11.

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
Thanks @dtf1337, for your contributions :)

**Comment by concentratedloss** [comment]
Thank you! It was a great experience for me. Sorry for all the fixes you
had to do!

Sincerely,
Bislam

**Pull request merged**